serverLib.sml

```diff
@@ -862,6 +889,11 @@ in
     in
       [a base_url "Overview", h3 (a jid (String.concat["Job ",jid])), pre (process jid s)]
     end
+  | req_body (DisplayQueue q) =
+    let
+      val _ = cgi_assert (List.exists (equal q) queue_dirs) 500 ["Unknown queue: ", q]
+    in a base_url "Overview" :: html_job_list (~1) (q, read_list q ())
```

Comment:
Using `~1` is not ideal, though it's ok. I'd probably have used an `option` instead.

Reply:
Good point, fixed